### PR TITLE
Post Effect resize fixes

### PR DESCRIPTION
**Pull request by GSterbrant**
### Description

A few more adjustments to the post effect resizing code, cleaning up some old deprecated stuff to do with render target scaling as well as removing the maxPixelRatio usage from the SSAO effect.

### CODE REVIEW

**Comment by lucaheft** [comment]
`this.width` and `this.height` in the bloom post-effect is never set, which results in the targets being resized every frame

**Review by slimbuck — APPROVED** [review]
Approving! (but pls also wait for @mvaligursky approval before merge).

**Review by mvaligursky — APPROVED** [review]

**Pull request merged**
